cli/src/cmd/forge/script/broadcast.rs

```diff
@@ -123,13 +128,111 @@ impl ScriptArgs {
                 })
                 .collect::<Result<Vec<_>>>()?;
 
+            let flashbots_sequence = deployment_sequence
+                .transactions
+                .iter()
+                .skip(already_broadcasted)
+                .filter(|tx| tx.bundle_block.is_some() && tx.bundle_gas.is_some())
+                .map(|tx_with_metadata| {
+                    let tx = tx_with_metadata.typed_tx();
+                    let from = *tx.from().expect("No sender for onchain transaction!");
+
+                    let kind = send_kind.for_sender(&from)?;
+                    let is_fixed_gas_limit = tx_with_metadata.is_fixed_gas_limit;
+
+                    let mut tx = tx.clone();
+
+                    tx.set_chain_id(chain);
+
+                    if let Some(gas_price) = self.with_gas_price {
+                        tx.set_gas_price(gas_price);
+                    } else {
+                        // fill gas price
+                        match tx {
+                            TypedTransaction::Eip2930(_) | TypedTransaction::Legacy(_) => {
+                                tx.set_gas_price(gas_price.expect("Could not get gas_price."));
+                            }
+                            TypedTransaction::Eip1559(ref mut inner) => {
+                                let eip1559_fees =
+                                    eip1559_fees.expect("Could not get eip1559 fee estimation.");
+                                let prio_fee = tx_with_metadata.bundle_gas.expect("Max priority fee per gas must be set");
+                                inner.max_fee_per_gas = Some(eip1559_fees.0.overflowing_add(prio_fee).0);
+                                inner.max_priority_fee_per_gas = tx_with_metadata.bundle_gas;
+                            }
+                        }
+                    }
+
+                    Ok((tx, tx_with_metadata.bundle_block, kind, is_fixed_gas_limit))
+                })
+                .collect::<Result<Vec<_>>>()?;
+
+            trace!(target: "script", "sequence: {:#?}", &sequence);
```

Comment:
I would say it's best to either:
1. remove this trace and just keep the full `transactions` trace on L90 or
2. remove the trace on L90 and add a trace for `flashbots_sequence` here.